doc/going-further/symfony-task.md

```diff
@@ -0,0 +1,34 @@
+# Symfony Task
+
+If castor lives near a Symfony application, you may want to use directly some
+commands. This is possible with the `AsSymfonyTask` attribute:
+
+```php
+#[AsCommand('hello', 'Says hello from a symfony application')]
+#[AsSymfonyTask(name: 'symfony:hello', console: [__FILE__])]
```

Comment:
The naming `AsSymfonyTask` makes totally sense in Castor code but that might be weird when you read you Symfony app code... a simple alias could fix this but maybe the doc could use one to simplify the discovery by newcomers? `BindAsCastorTask` for example?

Reply:
I Forgot the namespace in the doc, I added it. It's
```
use Castor\Attribute\AsSymfonyTask;
```

It's clearer with it, isn't?

Do you still think it must be changed?